.gitignore

```diff
@@ -1,3 +1,7 @@
+tests/Fixtures/app/cache/*
```

Comment:
it is more simple to just add a `.gitignore` in it, it will have the same role as `.gitkeep` but will avoid you to have to add it to the `.gitignore`